package.json

```diff
@@ -11,7 +11,7 @@
     "test:watch": "npm run test:raw -- -w",
     "commit": "git-cz",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
-    "check-coverage": "nyc check-coverage --lines 90 --functions 80 --branches 90",
+    "check-coverage": "nyc check-coverage --lines 70 --functions 70 --branches 70",
```

Comment:
Sorry I don't understand, if the failing code wasn't on this PR and there's no other PR open right now it means that code should be on master already. Right?